features/javascript-core.yml

```
@@ -0,0 +1,103 @@
+name: JavaScript core (initial support)
+description: Fundamental JavaScript language features, such as initial built-in objects, statements, and operators.
```

I think this description should attempt to describe what JavaScript _is_, then attempt to be transparent with our treatment of it. I'm thinking something like this instead:

```suggestion
description: JavaScript is a programming language that runs in browsers, usually through the `<script>` element. JavaScript has evolved over many years. This feature represents the oldest language features, such as built-in objects, statements, and operators. Also known as ECMAScript.
```